modular_skyrat/modules/customization/modules/mob/dead/new_player/preferences_setup.dm

```diff
@@ -41,14 +41,159 @@
 	if(randomise_flags & RANDOMIZE_FEATURES)
 		features = random_features()
 
-	//features = pref_species.get_random_features()
+	features = pref_species.get_random_features()
 	var/list/new_features = pref_species.get_random_features() //We do this to keep flavor text, genital sizes etc.
```

Comment:
It's already right under, I don't know why you're uncommenting it.
`features` doesn't seem to be used and gets overwritten by the following lines in the most part, so I'm not sure why this does any good.